tests/framework/cli/pipeline/test_pipeline_requirements.py

```diff
@@ -243,10 +184,8 @@ def test_no_requirements(
         self.call_pipeline_delete(fake_project_cli, fake_metadata)
         self.call_pipeline_pull(fake_project_cli, fake_metadata, fake_repo_path)
 
-        project_requirements_in = fake_repo_path / "src" / "requirements.in"
         project_requirements_txt = fake_repo_path / "src" / "requirements.txt"
```

Comment:
I know this is not your change, but we can probably remove this line while we're at it, since it's already defined on L179.
```suggestion
```